opentreemap/treemap/templates/instance_base.html

```diff
@@ -145,6 +156,7 @@
       <a class="btn btn-primary addBtn"
          data-action='addtree'
          data-feature="add_plot"
+         data-event-category="add-tree" data-event-action="start-add-tree"
```

Comment:
Probably we also want `data-event` atts on the top-level menu buttons (just below `instancetopnav`).